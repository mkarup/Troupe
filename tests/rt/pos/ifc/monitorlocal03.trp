let val p = spawn (fn () =>
                      let val _ = if true raisedTo `{secret}` then
                                    receive [ hn x => () ]
                                  else ()
                      in () end)
    val monRef = monitorlocal (p, `{secret}`)
    val _ = send (p, 42)
in
  rcvp (`{secret}`, [ hn x => x ])
end