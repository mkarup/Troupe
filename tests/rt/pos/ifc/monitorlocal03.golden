2024-12-11T12:25:44.608Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
>>> Main thread finished with value: ("DONE"@{secret}%{secret}, "d18af826-24ed-4c74-868a-16237330b382"@{secret}%{secret}, b570063e-9b88-42bf-b1af-9cf1801ec144@{secret}%{secret}, 0@{secret}%{secret})@{secret}%{secret}
